roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -242,6 +242,8 @@ kiali_defaults:
           expression:  "! healthy"
         - description: "Find: unknown nodes"
           expression:  "name = unknown"
+        - description: "Find: nodes with the 2 highest ranks"
```

Comment:
I still say this wording is going to create confusion among users. So I reserve my right to say "I told you so" in a few months when people start to ask "should I use < or > when testing rank?" :-p

Reply:
Fair enough, it is a little awkward to do `rank <= 2` to find the nodes with the top 2 ranks but what would be the alternative?

If ranking worked the opposite direction where higher ranks were a larger number you'd need to know the upper bound in order to find the "top ranking" nodes e.g. `rank > 99`.

Reply:
I don't know a solution, I just know "rank <= 2" looks confusing to me :) And I don't know why - because it is true when you see top 10 lists people say things like.. "And the number 1 song this week is <drum roll>... !!!" So there is experience here where the highest rank == 1. This is why I am not requiring any changes here.

I don't know a good substitute - "priority"... "preference".... "precedence" ... "degree" ...."distinction"... "primacy"... "stratum" I'm just looking at a thesaurus :)

Reply:
There is no good substitute.  A Rank of 1 is in general considered the best, or highest rank.  Feel free to say, "I told you so" :-) 

Having said that, one last shot at it:  "Find: nodes with the top 2 rankings".